presto-spark-base/src/main/java/com/facebook/presto/spark/execution/property/NativeExecutionSystemConfig.java

```diff
@@ -59,6 +59,9 @@ public class NativeExecutionSystemConfig
     // TODO: others use "-" separator and this property use _ separator. Fix them.
     private static final String HTTP_EXEC_THREADS = "http_exec_threads";
     private static final String NUM_IO_THREADS = "num-io-threads";
+    private static final String ASYNC_DATA_CACHE_ENABLED = "async-data-cache-enabled";
```

Comment:
Do we have these properties documented? If not, please, document them.

Reply:
Documented in where they are defined

Reply:
Where should I put it?